src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="/"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="navigation">
+        <ul class="nav__links">
+          <li>
+            <a
+              href="http://"
+              class="link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="http://"
+              class="link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Remember about correct indentation between parent and child elements. The elements inside the `header` should be indented to show they are children of the `header`.